<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depgraphV::RemoveRootDialog</class>
 <widget class="QDialog" name="depgraphV::RemoveRootDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Root Folder...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="rootFolders">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionSelect_None">
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Select None</string>
   </property>
  </action>
  <action name="actionInvert_Selection">
   <property name="text">
    <string>Invert Selection</string>
   </property>
   <property name="toolTip">
    <string>Invert Selection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>depgraphV::RemoveRootDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>depgraphV::RemoveRootDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_All</sender>
   <signal>triggered()</signal>
   <receiver>rootFolders</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_None</sender>
   <signal>triggered()</signal>
   <receiver>rootFolders</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_All</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::RemoveRootDialog</receiver>
   <slot>_updateEnabledFlags()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelect_None</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::RemoveRootDialog</receiver>
   <slot>_updateEnabledFlags()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rootFolders</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>depgraphV::RemoveRootDialog</receiver>
   <slot>_updateEnabledFlags()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInvert_Selection</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::RemoveRootDialog</receiver>
   <slot>_invertSelection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInvert_Selection</sender>
   <signal>triggered()</signal>
   <receiver>depgraphV::RemoveRootDialog</receiver>
   <slot>_updateEnabledFlags()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_updateEnabledFlags()</slot>
  <slot>_invertSelection()</slot>
 </slots>
</ui>
